config/routes.rb

```diff
@@ -193,6 +193,8 @@
     get '/rules_of_use' => 'users/rules_of_use#new'
     post '/rules_of_use' => 'users/rules_of_use#create'
 
+    delete '/openid_connect/logout' => 'openid_connect/logout#delete'
```

Comment:
Should this route be defined with the other `/openid_connect/` routes? Based on its placement here, I expect this would be a localizable route, whereas the others aren't. Is that intentional?

https://github.com/18F/identity-idp/blob/db85fd9e3ec88866f2fc238eee428fdaf637db78/config/routes.rb#L32-L33

Reply:
I initially had it here since I wanted it limited to authenticated users, but that should be covered by other logic.